src/wallet/mod.rs

```diff
@@ -1683,6 +1690,15 @@ pub(crate) mod test {
     use crate::signer::{SignOptions, SignerError};
     use crate::wallet::AddressIndex::{LastUnused, New, Peek, Reset};
 
+    fn create_drain_all_and_consolidate_psbt(
```

Comment:
any better name for this test helper function?(drain all and send back to itself?)

Reply:
I think this function might come handy when working on tests, but if we introduce it we should use it for everything, not just for `test_do_not_remove_partial_sigs_after_finalize_sign_option` and `test_do_not_finalize_sign_option`.
OTOH, it's just 4 lines of code, so I don't think it's a big deal if we go on copy-pasting those instead of having a function

If you decide to use this function in every test, I agree that it needs a better name, but I don't have any better ideas...

Reply:
I didn't use it for existing test cases to avoid too much code changes. But I am happy to do so if other maintainer don't mind? I can also remove it and just copy and paste. will wait until having more opinion from others first

Reply:
Since no one is really chiming in, I'd say we can drop this function and copy-paste its four lines of code - so we don't have to find a name for it and we keep the code consistent

Reply:
No worries 😁